apps/condo/pages/tour/guide.tsx

```diff
@@ -188,21 +188,22 @@ const AboutAppBlock = () => {
                                     }
                                 </Col>
                                 {
-                                    get(guideModalCardLink, [locale, type]) && (
+                                    locale === 'ru' && modalCardExamples[type] && (
                                         <Col xs={24} md={12}>
                                             <Card title={(
                                                 <img
-                                                    src={`/onboarding/guide/aboutApp/${locale}/${type}/modalLogo.webp`}
+                                                    src={modalCardExamples[type].imageUrl}
                                                     style={CARD_IMAGE_STYLES}
                                                 />
                                             )}
                                             >
                                                 <Card.CardBody
-                                                    description={intl.formatMessage({ id: `tour.guide.aboutApp.${type}.modal.card.body` as FormatjsIntl.Message['ids'] })}
+                                                    description={modalCardExamples[type].text}
```

Comment:
text without translate?

Reply:
@toplenboren Told me that this translation should have been in the env var as well

Reply:
Whole card should be hidden if there is no translate